### DISCUSSION TITLE
Unable to set max height in Prism.Tabs component

### DISCUSSION THREAD

**Discussion by BenJeau** [mantinedev/mantine#3173] [Q&A]
Hi,

I've dug in the discussions and issues, found these related issues, but not specifically for the `<Prism.Tabs />` component:
* https://github.com/mantinedev/mantine/issues/724
* https://github.com/mantinedev/mantine/discussions/1252

I'm able to set the max height for the prism component if its in a regular `<Tab />` component and when the component is alone, but not with `<Prism.Tabs />`

Here's my codesandbox link https://codesandbox.io/s/polished-fire-ifuxn1?file=/src/App.tsx

**Comment by rtivital** [comment]
You should build a custom component for that, it will be difficult to implement with Prism.Tabs

**Reply by BenJeau** [reply]
Okk, I think I'll just use the Tabs component as its a very similar component, thanks :)
